docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,135 @@
+# Release 369 (19 Jan 2022)
+
+## General
+
+* Add support for `Pacific/Kanton` time zone. ({issue}`10679`)
+* Display `Physical input read time` using most succinct time unit in `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`10576`)
+* Fine tune request retry mechanism in HTTP event listener. ({issue}`10566`)
+* Add support for using PostgreSQL and Oracle with database backed resource
+  groups. ({issue}`9812`)
+* Remove unnecessary fine-grained spill toggles. ({issue}`10591`)
```

Comment:
Link to docs or list which ones are no longer supported.

Reply:
Done